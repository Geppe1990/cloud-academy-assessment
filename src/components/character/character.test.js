import React from "react";
import Character from "./character";

jest.mock("react-router-dom", () => ({
	...jest.requireActual("react-router-dom"),
	useParams: () => ({
		id: "1"
	})
}));

it("Renders character without crashing", () => {
	shallow(<Character />);
});

it('match the snapshot', () => {
	const wrapper = renderer.create(<Character />).toJSON();
	expect(wrapper).toMatchSnapshot();
});