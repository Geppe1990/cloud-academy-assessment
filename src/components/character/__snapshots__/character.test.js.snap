// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match the snapshot 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col-sm-12"
    >
      <span>
        Loading...
      </span>
    </div>
  </div>
</div>
`;
